city_furniture/views/wfs/furniture_signpost.py

```diff
@@ -26,7 +26,16 @@
     ),
     DescribedFeatureField("location", description="Signpost's location (point) in EPSG:3879 coordinates."),
     DescribedFeatureField(
-        "location_name", description="Verbose name for the signpost's location, e.g. street, park or island."
+        "location_name_fi",
+        description="Verbose name for the signpost's location, e.g. street, park or island in finnish.",
```

Comment:
Language names in English are capitalized -> `Finnish`. Same applies to others :point_down: